cloudflare/src/framework/apiclient.rs

```diff
@@ -1,9 +1,10 @@
-//! This module contains the synchronous (blocking) API client.
+//! This module contains the API for a client.
```

Comment:
Self-comment:  I've modified this file to solely contain the traits for the sync and async clients.  That way I moved the implementations to separate files so that I can cleanly elide them with a cfg macro over the reqwest-http feature.